app/assets/javascripts/game.js

```diff
@@ -0,0 +1,77 @@
+$(function() {
+    // Picks up all the latest positions from
+    // the database
+    var currentPositions = $('.piece_pos').data('positions');
+
+    $.each(currentPositions, function(key, value) {
+      var currentCell = 'td[data-x="' + value.x_position + '"]' +
+                        '[data-y="' + value.y_position + '"]';
+      //formula to map DB pieace IDs to the IDs in view
+      var pieceIdOnBoard = value.id % 32;
+      $('#' + pieceIdOnBoard).appendTo(currentCell);
+    });
+
+    // Makes all cells that have <i> elements
+    // draggble. If move is invalid, the piece will
+    // return to its cell
+    $('#chess-board td i').draggable({
+      containment: "#chess-board",
+      cursor: "all-scroll",
+      opacity: 0.6,
+      revert: 'invalid'
+    });
+
+    // Each cell will accept only <i> elements.
+    // Only when pointer hovers over the cell
+    // the drop will occur.
+    $('#chess-board td').droppable( {
+      // white cannot be dropped on white and vice versa
+      // this logic can be removed when move! method works
```

Comment:
I was commenting on this comment ` // this logic can be removed when move! method works` and am referring to the plan of removing it